modules/user/src/main/Plan.scala

```diff
@@ -18,7 +18,6 @@ case class Plan(
 
   def enable = copy(
     active = true,
-    months = months max 1,
```

Comment:
why remove the garanty that months >= 1 ?

Reply:
At some point I had a failing test and mistakenly removed this guarantee.